src/tests/integration/test-data/show/AddConst.applyOp(uint256).cse.expected

```diff
@@ -1,64 +1,32 @@
 
-┌─ 1 (root, split, init)
```

Comment:
I think this should still be a `split`.

Reply:
Yup. This happened because I stored the fields as `#buf`s, which are of sort `Bytes`, and the `#lookup(Map,Int)` rule that reads the storage expects the value to be an `Int`.